libs/designer/src/lib/core/state/templates/manifestSlice.ts

```diff
@@ -7,6 +7,23 @@ import type { FilterObject } from '@microsoft/designer-ui';
 export const templatesCountPerPage = 25;
 const initialPageNum = 0;
 
+interface ContentInfo<T> {
+  value: T;
+  isEditable?: boolean;
+}
+
+export interface ViewTemplateDetails {
+  id: string;
+  basicsOverride?: Record<
```

Comment:
This only applies to non accesslator template? Do we want to make this a bit more future proof, or maybe there's some other story in mind?

Reply:
this applies to all templates even accelerators, we can specify which workflow to override here